pypesto/petab/importer.py

```diff
@@ -280,7 +281,7 @@ def create_model(
             logger.info(
                 f"Compiling amici model to folder " f"{self.output_folder}."
             )
-            self.compile_model(**kwargs)
+            self.compile_model(validate=self.validate_petab, **kwargs)
```

Comment:
This fails for pysb models. Pass `validate` only if `from petab.models import MODEL_TYPE_SBML; petab_problem.model.type_id == MODEL_TYPE_SBML`.